tests/integration/test.js

```diff
@@ -190,3 +188,51 @@ describe('Mugshot integration with disabled acceptFirstBaseline', function() {
     return wdioInstance.end();
   });
 });
+
+
+describe('Mugshot integration with recursive rootDirectory', function() {
```

Comment:
Why is this separate? Does the setup differ between the 2? If so, let's create a separate module for this. If they don't differ, please create a nested `describe` block under the first `Mugshot integration` one.